src/main/webapp/jsp/instructorSearch.jsp

```diff
@@ -1,478 +1,36 @@
-                                                Comment on <%=PageData.sanitizeForHtml(student.name)%>
-                                            </a>
-                                        </li>
-                                        <li role="presentation">
-                                            <a role="menuitem" tabindex="-1" href="<%=data.getCourseStudentDetailsLink(student.course, student)
-                                                                                      +"&"+ Const.ParamsNames.SHOW_COMMENT_BOX+"=team"%>">
-                                                Comment on <%=PageData.sanitizeForHtml(student.team)%>
-                                            </a>
-                                        </li>
-                                        <% if (!student.section.equals(Const.DEFAULT_SECTION)) { %>
-                                               <li role="presentation">
-                                                   <a role="menuitem" tabindex="-1" href="<%=data.getCourseStudentDetailsLink(student.course, student)
-                                                                                             +"&"+ Const.ParamsNames.SHOW_COMMENT_BOX+"=section"%>">
-                                                       Comment on <%=PageData.sanitizeForHtml(student.section)%>
-                                                   </a>
-                                               </li>
-                                        <% } %>
-                                    </ul>
-                                </div>
-                            </td>
-                        </tr>                           
-                <%
-                    }
-                %>
-                                        </tbody>
-                                    </table>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-             <% } %>
-        </div>
-    </div>
-
-    <jsp:include page="<%=Const.ViewURIs.FOOTER%>" />
-</body>
-</html>
\ No newline at end of file
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/instructorSearch" prefix="search" %>
+
+<c:set var="jsIncludes">
+    <script type="text/javascript" src="/js/instructor.js"></script>
+    <script type="text/javascript" src="/js/instructorSearch.js"></script>
+    <script type="text/javascript" src="/js/additionalQuestionInfo.js"></script>
+</c:set>
+
+<ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Search" jsIncludes="${jsIncludes}">
+
+    <search:searchPageInput />
+    <t:statusMessage />
+    
+    <c:if test="${data.commentSearchResultBundle.resultSize != 0}">
```

Comment:
The empty expression returns true only for null and empty String/array/List/Map/Collection, otherwise it is false. CommentResultSearchBundle is a class, not a String/array/List/Map/Collection. As long as other attributes in the class are not empty, 'empty' will return false.

Also, the if condition above does not check whether the CommentSearchResultBundle is empty. It checks whether a field containing number of results is 0. Since it is a custom empty-checking  condition, 'empty' will always return false.